src/jest.config.ts

```diff
@@ -71,11 +71,7 @@ const config: InitialOptionsTsJest = {
 
   // A set of global variables that need to be available in all test environments
   // NOTE: these must be JSON serializable
-  globals: {
```

Comment:
This is needed  because of a deprecation: https://github.com/kulshekhar/ts-jest/blob/main/CHANGELOG.md#deprecations